src/std/math.ab

```diff
@@ -29,6 +29,6 @@ pub fun ceil(number: Num): Num {
 /// Returns the absolute value of a number
 #[allow_absurd_cast]
 pub fun abs(number: Num): Num {
-    if number < 0: return -number
+    if number < 0: return number * -1
```

Comment:
What's the point of this? Why not leave the `-number`?

Reply:
because doesn't work with this change, without that change this test fails.